releases/v0.9.0-alpha0.md

```diff
@@ -0,0 +1,78 @@
+# Release Notes for v0.9.0-alpha
+
+Release Date: April 25th, 2024
+
+This is our first release based on Kata main, but it is an alpha release that supports
+only a subset of features.
+
+This release is based on [3.4.0](https://github.com/kata-containers/kata-containers/releases/tag/3.4.0) of Kata
```

Comment:
Hi @fitzthum !

The Kata teams recommends to use full project name "Kata Containers" when possible.